doc/client_lua_api.md

```diff
@@ -770,7 +773,7 @@ Methods:
         ```
 
 * `get_last_look_horizontal()`:
-    * returns last look horizontal angle
+    * returns last look horizontal angle in radians (can be lower than 0 or higher than 2*pi if the player has made whole turns on himself, increases counterclockwise and 0° is East)
```

Comment:
I think we should probably fix this by inserting appropriate `fmod`s rather than documenting it.

Reply:
It should at least be mentioned that the returned angle is in radians, as is the case in the server-side API (which unfortunately I only discovered lately)